### PR TITLE
LPS-123261 Change log levels

### PR DESCRIPTION
**Pull request by robertoDiaz**

### CODE REVIEW

**Comment by robertoDiaz** [comment]
ci:test:sf

**Comment by robertoDiaz** [comment]
ci:test:stable

**Comment by robertoDiaz** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
Just started reviewing :)

File: portal-impl/src/com/liferay/portlet/documentlibrary/util/PDFProcessorImpl.java [hunk 1]
```diff
@@ -654,177 +654,207 @@ private void _generateImagesPB(FileVersion fileVersion, File file)
...
-						" resulted in a canceled timeout for " + future;
+					liferayConverter.generateImagesPB();
 				}
-
-				_log.error(errorMessage);
-
-				_fileVersionPreviewEventListener.onFailure(fileVersion);
-
+			}
+			catch (TimeoutException timeoutException) {
```
Comment (adolfopa) [inline portal-impl/src/com/liferay/portlet/documentlibrary/util/PDFProcessorImpl.java:812]: @robertoDiaz Previously, this exception was handled by the general Exception catch. Why do we need to propagate without logging anything now?

**Comment by adolfopa** [comment]
ci:forward

**Pull request closed without merging**